kodilla-jdbc/src/test/java/com/kodilla/jdbc/StoredProcTestSuite.java

```diff
@@ -0,0 +1,64 @@
+package com.kodilla.jdbc;
+
+import org.junit.Test;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static java.lang.Boolean.FALSE;
+import static org.junit.Assert.assertEquals;
+
+public class StoredProcTestSuite {
+    @Test
+    public void testUpdateVipLevels() throws SQLException{
+        //Given
+        DbManager dbManager = DbManager.getInstance();
+        String sqlUpdate = "UPDATE READERS SET VIP_LEVEL=\"Not set\"";
+        Statement statement = dbManager.getConnection().createStatement();
+        statement.executeUpdate(sqlUpdate);
+
+        //When
+        String sqlProcedureCall = "CALL UpdateVipLevels()";
+        statement.execute(sqlProcedureCall);
+        //Then
+        String sqlCheckTable = "SELECT COUNT(*) AS HOW_MANY FROM READERS WHERE VIP_LEVEL=\"Not set\"";
+        ResultSet rs = statement.executeQuery(sqlCheckTable);
+        int howMany = -1;
+        if (rs.next()){
+            howMany = rs.getInt("HOW_MANY");
+        }
+        assertEquals(0,howMany);
+    }
+
+    @Test
+    public void testUpdateBestsellers() throws SQLException{
+        //Given
+        DbManager dbManager = DbManager.getInstance();
+        String sqlUpdate = "UPDATE BOOKS SET BESTSELLER=null";
+        Statement statement = dbManager.getConnection().createStatement();
+        statement.executeUpdate(sqlUpdate);
+
+        //When
+        String sqlProcedureCall = "CALL UpdateBestsellers()";
+        statement.execute(sqlProcedureCall);
+
+        //Then
+        String sqlCheckTable = "SELECT COUNT(*) AS HOW_MANY_BTS FROM BOOKS WHERE BESTSELLER=TRUE";
+        ResultSet rs = statement.executeQuery(sqlCheckTable);
+        int howManyBestsellers = -1;
```

Comment:
Masz tego typu fragmenty używane w dwóch miejscach. Na przyszłość jakbyś pisał więcej testów to pewnie tego typu rzeczy by się powtarzały. Wtedy lepiej utworzyć sobie metodę pomocniczą, przyjmującą `ResultSet` oraz nazwę pola, które chcesz wyciągnąć. 
```
int howManyBestsellers = -1;
        if (rs.next()){
            howManyBestsellers = rs.getInt("HOW_MANY_BTS");
        }
```
Zmieniłoby się na wywołanie metody: 
```
int howManyBestsellers = getInt(rs, "HOW_MANY_BTS")
```
Byłoby to dużo bardziej czytelne. 
Analogicznie mógłbyś dodać kolejne metody `getBoolean`, `getDouble` itd.